src/components/books/books/inputBook.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../../redux/books/books';
+
+const InputBook = () => {
+  const dispatch = useDispatch();
+
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+
+  const inputTitleHandler = (e) => {
+    setTitle(e.target.value);
+  };
+
+  const inputAuthorHandler = (e) => {
+    setAuthor(e.target.value);
+  };
+
+  const submitBookToStore = () => {
+    const newBook = {
+      id: uuidv4(),
+      title: title.trim(),
+      author: author.trim(),
+    };
+    dispatch(addBook(newBook));
+  };
```

Comment:
[OPTIONAL] You can "clean" the value of your inputs after you submit your form by setting your `title` and `author` state to an empty string. And using this value on your inputs

e.g.
```
const submitBookToStore = () => {
    const newBook = {
      id: uuidv4(),
      title: title.trim(),
      author: author.trim(),
    };
    dispatch(addBook(newBook));
    setTitle('');
    setAuthor('');
  };
```

```
...
return (
    <form>
      <input onChange={inputTitleHandler} value={title} type="text" placeholder="title" />
      <input onChange={inputAuthorHandler} value={author} type="text" placeholder="author" />
      <button type="button" onClick={submitBookToStore}>
        Submit
      </button>
    </form>
  );
```